pkg/generator/github/github.go

```diff
@@ -129,6 +129,13 @@ func (g *Generator) generate(
 		return nil, nil, fmt.Errorf("error decoding response: %w", err)
 	}
 
+	if resp.StatusCode >= 300 {
+		if message, ok := gat["message"]; ok {
+			return nil, nil, fmt.Errorf("error generating token: response code: %d, response: %v", resp.StatusCode, message)
+		}
+		return nil, nil, fmt.Errorf("error generating token: response code: %d, response: %v", resp.StatusCode, gat)
```

Comment:
do not put the entire struct in the log. There could still be sensitive information in there that is logged.